license/license.md

```diff
@@ -47,3 +47,22 @@ Here is a handy table that explains the events leading up to and after license e
 | Expiration | A notification indicating that the license is expired, 14 day grace period begins | T - 0 |
 | Ingest Disabled | Ingest is disabled and a notification indicating that the license is expired | T + 15 days |
 | Query Disabled | Searching is disabled and a notification indicating that the license is expired | T + 30 days |
+
+## Gravwell License Types
+
+| Type                | Identifier  | Basic Features | Unlimited Ingest | Cluster | Replication | CBAC | HA Webservers | SSO | AI Logbot | Notes                                               |
+|---------------------|-------------|:--------------:|:----------------:|:-------:|:-----------:|:----:|:-------------:|:---:|:---------:|:----------------------------------------------------|
+| Free                | UNLICENSED  | ✅             |                  |         |             |      |               |     |           | 2GB/day ingest, no licensing required, non-commercial use only. |
+| Community Edition   | community   | ✅             |                  |         |             |      |               |     | 🗸         | 14.95 GB/day ingest, authorized for commercial use. |                               
```

Comment:
Is the Community Edition still free? We may need to make that more obvious.

Reply:
yes, i'll adjust the wording, maybe drop a link to community edition?